src/main/java/org/jitsi/xmpp/mucclient/MucClient.java

```diff
@@ -16,23 +16,18 @@
  */
 package org.jitsi.xmpp.mucclient;
 
-import org.jivesoftware.smack.AbstractXMPPConnection;
-import org.jivesoftware.smack.ConnectionListener;
-import org.jivesoftware.smack.ReconnectionManager;
-import org.jivesoftware.smack.XMPPConnection;
-import org.jivesoftware.smack.iqrequest.IQRequestHandler;
-import org.jivesoftware.smack.packet.Stanza;
-import org.jivesoftware.smack.tcp.XMPPTCPConnection;
-import org.jivesoftware.smack.tcp.XMPPTCPConnectionConfiguration;
-import org.jivesoftware.smackx.muc.MultiUserChat;
-import org.jivesoftware.smackx.muc.MultiUserChatManager;
-import org.jivesoftware.smackx.ping.PingManager;
-import org.jxmpp.jid.EntityBareJid;
-import org.jxmpp.jid.parts.Resourcepart;
+import org.jivesoftware.smack.*;
+import org.jivesoftware.smack.iqrequest.*;
+import org.jivesoftware.smack.packet.*;
+import org.jivesoftware.smack.tcp.*;
+import org.jivesoftware.smackx.muc.*;
+import org.jivesoftware.smackx.ping.*;
+import org.jxmpp.jid.*;
+import org.jxmpp.jid.parts.*;
```

Comment:
And why we decided to change the code convention?
At least there should be a document for this specific code convention the same way we have for the rest of the java code: https://desktop.jitsi.org/Documentation/CodeConvention

And now let's complain about this a little:
It is very inconvenient to have different code convention for the same language, as IDEs (at least IDEA) doesn't have code convention per module and it is general for the whole project. This means I need to open new window just for this project because it uses different code convention. 
When we start depending on another jitsi projects like jicoco, I need to open jicoco twice in both windows, and I need to not forget that I need to debug in window2 where it is jibri, but write code in window1 for jicoco, with the rest of the projects, because of same style convention. There are many inconveniences that come with this change. 
If we decide we are changing code convention, let's do it globally for all projects so we do not fall in this inconvenient cases.

Reply:
fixed, it by the way so we can merge, but it took two force pushes because of the IDE code convention settings, which I cannot change because of the other projects.

Reply:
yeah, i think it is fair to keep it separated by language.  i always have different intellij windows open for the different projects so i didn't think per-project was a big deal.  we can make the change to the imports in the java file as part of this change if you want.